tasks/install.yml

```diff
@@ -35,6 +35,17 @@
     - "{{ prometheus_config_dir }}/rules"
     - "{{ prometheus_config_dir }}/file_sd"
 
+- name: create prometheus console directories
```

Comment:
`remote_src` copy operation only works properly if the folders already exists on the remote source.